src/core/components/AuthenticateButton/index.js

```diff
@@ -53,9 +55,17 @@ export const mapStateToProps = (state) => ({
 });
 
 export const mapDispatchToProps = (dispatch) => ({
-  handleLogOut() {
+  handleLogOut({ api }) {
+    const { location } = window;
+    if (config.get('apiHost') === `${location.protocol}//${location.host}`) {
```

Comment:
Removing this will mean that logging out doesn't work when you aren't using the proxy server. If we always use the proxy then that would be fine, but it could be a bit annoying as it is. Of course you can just use dev tools to clear the cookies in development so that may not be a big deal.